bluephos/modules/filter.py

```diff
@@ -0,0 +1,18 @@
+import pandas as pd
+
+
+# filter.py
+class FilterTask:
```

Comment:
I don't really see the need for a class that stores data, seems to me all you need is a single func

```py
def filter(df: pd.DataFrame, column: str, threshold: float, filter_in=True) -> pd.DataFrame:
```

with the bit of logic you have now in `task`, no real need to implement here and have a helper in the tasks file, really only need the one.